mage_ai/frontend/components/datasets/overview/utils.ts

```diff
@@ -15,14 +17,14 @@ export function createMetricsSample(statistics) {
 
   stats.map((key) => {
     if (METRICS_KEYS.includes(key)) {
-      let value = statistics[key].toPrecision(2);
-      let bar = [false];
+      let bar = [false, 0];
+      let value = statistics[key];
       const order = HUMAN_READABLE_MAPPING[key];
       const index = METRICS_SORTED_MAPPING[key];
       if (PERCENTAGE_KEYS.includes(key)) {
-        value *= 100;
-        bar = [true, value];
-        value = `${value}%`;
+        value = value.toPrecision(2);
+        bar = [true, (value * 100)];
+        value = getPercentage(value);
```

Comment:
should this be calling the `getPercentage` method on the original `statistics[key]` value instead of the value modified with `toPrecision`?

Reply:
For the original values that are not percentage keys we don't want to turn them into percentages. That's why we only call get percentage here. So we avoid values like 0.0 when it's 0 or values with e+3 when it's in the thousands.